lms/djangoapps/verify_student/urls.py

```diff
@@ -23,6 +23,16 @@
         }
     ),
 
+    # This is for A/B testing.
+    url(
+        r'^start-flow_ab/{course}/$'.format(course=settings.COURSE_ID_PATTERN),
```

Comment:
The URL itself shouldn't give away that this is part of an A/B test. I'd suggest using something like "begin-flow" instead.